Wikipedia/UI-V5/WMFArticlePreviewCell.m

```diff
@@ -24,6 +30,18 @@ - (void)prepareForReuse {
     self.summaryLabel.text = nil;
 }
 
+-(void)awakeFromNib {
+    [self rememberSettingsFromIB];
+}
+
+-(void)rememberSettingsFromIB {
+    NSRange r = NSMakeRange(0, 1);
+    self.summaryLabelAttributesFromIB = [self.summaryLabel.attributedText attributesAtIndex:0 effectiveRange:&r];
+    self.paddingAboveDescriptionFromIB = self.paddingConstraintAboveDescription.constant;
```

Comment:
@coreyfloyd active = no isn't what we want i think - we still want this constrain - we just don't want the padding.